plugins/base64/unit_test/test_encode.py

```diff
@@ -0,0 +1,15 @@
+from unittest import TestCase
+from komand_base64.actions.encode import Encode
+import logging
+
+
+class TestEncode(TestCase):
```

Comment:
I don't think we want to include this file in the merged PR

Reply:
I disagree. It's the plugin we use to start all other plugins (it's a good starter one anyway). So I like having the model for unit tests go out with this PR.